server/application/application.go

```diff
@@ -588,7 +610,7 @@ func (s *Server) Get(ctx context.Context, q *application.ApplicationQuery) (*app
 	// We must use a client Get instead of an informer Get, because it's common to call Get immediately
 	// following a Watch (which is not yet powered by an informer), and the Get must reflect what was
 	// previously seen by the client.
-	a, err := s.getApplicationEnforceRBACClient(ctx, rbacpolicy.ActionGet, appNs, appName, q.GetResourceVersion())
+	a, err := s.getApplicationEnforceRBACClient(ctx, rbacpolicy.ActionGet, q.GetAppProject(), appNs, appName, q.GetResourceVersion())
```

Comment:
Should we be using `GetProject` if `Project` was specified instead of `AppProject` in the request?

Reply:
Changed to eliminate the `appProject` field and instead rely on the existing `project` and `projects` field, using same order of precedence as other endpoints. Thanks for catching it!